akka-stream/src/main/mima-filters/2.5.10.backwards.excludes

```diff
@@ -0,0 +1,2 @@
+# #24604 Deduplicate logic for IODispatcher
+ProblemFilters.exclude[MissingTypesProblem]("akka.stream.ActorAttributes$Dispatcher$")
```

Comment:
Oh, actually, we had to make a release yesterday because of a regression, so this file should now be `2.5.11.backwards.excludes`

Reply:
thanks, fixed